scripts/table_diff/changes/changes.go

```diff
@@ -105,19 +126,33 @@ func getColumnChanges(file *gitdiff.File, table string) (changes []change) {
 				})
 			}
 
-			if addedColumn.pk && !deletedColumn.pk {
+			if addedColumn.pk() && !deletedColumn.pk() {
 				changes = append(changes, change{
 					Text:     fmt.Sprintf("Table %s: primary key constraint added to column %s", backtickStrings(table, deletedName)...),
 					Breaking: false,
 				})
 			}
 
-			if !addedColumn.pk && deletedColumn.pk {
+			if !addedColumn.pk() && deletedColumn.pk() {
 				changes = append(changes, change{
 					Text:     fmt.Sprintf("Table %s: primary key constraint removed from column %s", backtickStrings(table, deletedName)...),
 					Breaking: false,
 				})
 			}
+
+			if addedColumn.incremental() && !deletedColumn.incremental() {
+				changes = append(changes, change{
+					Text:     fmt.Sprintf("Table %s: incremental sync support added to column %s", backtickStrings(table, deletedName)...),
```

Comment:
```suggestion
					Text:     fmt.Sprintf("Table %s: column %s added to cursor for incremental syncs", backtickStrings(table, deletedName)...),
```

I think it was fine, but this might be a little clearer